packages/example/pages/_app.tsx

```diff
@@ -155,6 +155,7 @@ const config = getDefaultConfig({
       ],
     },
   ],
+  ssr: true,
```

Comment:
@KosmosKey We might need to reference this in our docs too since it can be passed through `getDefaultConfig`